include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
Could you please add ```#pragma warning( disable : 4521 )``` to your project only for now? I would better open an issue and investigate it whenever we have VS2015 installed.